addons/survey/views/survey_templates_management.xml

```diff
@@ -78,11 +78,13 @@
     <!-- ============================================================ -->
 
     <template id="survey_button_form_view" name="Survey: back to form view">
-        <div groups="survey.group_survey_manager" t-ignore="true" class="alert alert-info alert-dismissible rounded-0 fade show d-print-none css_editable_mode_hidden mb-0">
-            <div t-ignore="true" class="text-center">
-                <a t-attf-href="/web#view_type=form&amp;model=survey.survey&amp;id=#{survey.id}&amp;action=survey.action_survey_form"><i class="fa fa-fw fa-arrow-right"/><span t-if="answer and answer.test_entry">This is a test survey. </span>Edit Survey</a>
+        <div name='blue_header'>
```

Comment:
I think this div is not necessary. Actually you could probably simply add a class to the current div and replace it (a bit like you did, but without adding yet another div hierarchy). I would also avoid "blue_header" naming as it is not standard nor really helpful. Could be something like ``o_survey_manage``, having both o_survey namespacing and manage keyword telling its purpose is to manage surveys. What do you think ?

Reply:
First of all, thanks for a review ^^ And agreed, the name is not the best, I will change it. I added div with the following reasoning. The way you proposed with class, it was done before as well - "//div[hasclass('alert')]/div" and it works when the user has survey user rights. But if the user is not a survey user, if I am not mistaken, we won't see any of the classes from the current label because - <div groups="survey.group_survey_manager" t-ignore="true" class="alert alert-info......>. In hr_appraisal_survey, the user  without survey rights still can see the answers of the feedback survey, and makes sense that he will also see 'back to appraisal' button. What do you think? Can I still grab the blue_head (the name to be changed)  when the user does not belong to group_survey_manager, without adding extra div?

Reply:
You may remove the groups attribute in inheriting template, normally xpath allows you to do it ?

Reply:
Sure but if I have xpath expr="//div[hasclass('alert')]/div" and I log in with the user that does not belong to "survey.group_survey_manager", the xpath won't work, because I won't see the alert class to start with, no?